app/src/main/java/com/epfl/beatlink/ui/library/PlaylistOverview.kt

```diff
@@ -0,0 +1,165 @@
+
+                    // Playlist details
+                    Column(
+                        modifier = Modifier.weight(1f).fillMaxHeight(),
+                        verticalArrangement = Arrangement.SpaceBetween) {
+                          Text(
+                              text = selectedPlaylistState.playlistName,
+                              style = TypographyPlaylist.headlineLarge,
+                              color = MaterialTheme.colorScheme.primary,
+                              modifier = Modifier.testTag("playlistTitle"))
+                          Spacer(modifier = Modifier.height(4.dp))
+                          IconWithText(
+                              "@" + selectedPlaylistState.playlistOwner,
+                              "ownerText",
+                              Icons.Outlined.AccountCircle,
+                              TypographyPlaylist.headlineMedium)
+                          IconWithText(
+                              selectedPlaylistState.playlistCollaborators.joinToString(", "),
+                              "collaboratorsText",
+                              collab,
+                              TypographyPlaylist.headlineSmall)
+                          IconWithText(
+                              if (selectedPlaylistState.playlistPublic) "Public" else "Private",
+                              "publicText",
+                              Icons.Outlined.Lock,
+                              TypographyPlaylist.headlineSmall)
+                          Spacer(modifier = Modifier.height(10.dp))
+                          ViewDescriptionButton {}
+                        }
+                  }
+              Spacer(modifier = Modifier.height(16.dp))
+              FilledButton(
+                  "Add to this playlist",
+                  "addToThisPlaylistButton") { /* Opens a page to add songs */}
+              Spacer(modifier = Modifier.height(16.dp))
+              PrincipalButton(
+                  "Export this playlist", "exportButton") { /* Exports the playlist to Spotify */}
+              Spacer(modifier = Modifier.height(16.dp))
+              if (selectedPlaylistState.nbTracks == 0) {
+                Text(
+                    text = "NO SONGS ADDED",
+                    style = TypographyPlaylist.displayMedium,
+                    modifier = Modifier.padding(top = 165.dp).testTag("emptyPlaylistPrompt"))
+              } else {
+                Box(modifier = Modifier.fillMaxSize().heightIn(min = 0.dp, max = 400.dp)) {
+                  LazyColumn(
+                      verticalArrangement = Arrangement.Top,
+                      contentPadding = PaddingValues(horizontal = 16.dp, vertical = 16.dp),
+                      modifier = Modifier.fillMaxSize()) {
+                        // List of tracks
+                        items(1) { trackId ->
+                          // val track = playlistViewModel.getTrackById(trackId)
+                          TrackVoteCard(sample)
```

Comment:
This was just a test to display the TrackVoteCard() composable, because the logic with track ids has not been implemented yet. Indeed, the track ids will be retrieved from `selectedPlaylistState` once the list of track ids will not be empty.